mime-types/Network/Mime.hs

```diff
@@ -494,9 +499,9 @@ defaultMimeMap = Map.fromAscList [
     , ("m2a", "audio/mpeg")
     , ("m2v", "video/mpeg")
     , ("m3a", "audio/mpeg")
-    , ("m3u", "audio/x-mpegurl")
+    , ("m3u", "audio/mpegurl")
     , ("m3u8", "application/vnd.apple.mpegurl")
```

Comment:
Maybe turn this into `audio/mpegurl` as well? (since `.m3u` and `.m3u8` are the same thing)

Reply:
Oh, good point. I noticed that only `application/vnd.apple.mpegurl` has been registered by IANA, so I used that instead of `audio/mpegurl`.